.github/workflows/lint.yaml

```diff
@@ -0,0 +1,38 @@
+name: lint
+
+on:
```

Comment:
why not on all PRs?

Reply:
no reason not to 14d0a3a04125e08b0a30e10ae1fc9deeb4b113c9